src/CacheInterface.php

```diff
@@ -2,12 +2,26 @@
 
 namespace React\Cache;
 
+use React\Promise\PromiseInterface;
+
 interface CacheInterface
 {
-    // @return React\Promise\PromiseInterface
+    /**
+     * @param $key
+     * @return PromiseInterface
+     */
     public function get($key);
 
+    /**
+     * @param $key
+     * @param $value
```

Comment:
If they don't exist, they're out of scope, yes. But in that case, those `@param`s are just noise and can be deleted.